### ISSUE TITLE
Image doesn't load when app is reopening

### ISSUE THREAD

**Issue by prokhorovxo** [VODGroup/VoiceOverDesigner#196]
I found some rather strange behavior. 

**Steps to reproduce:**
1. Run the app from Xcode
2. Create a new project, add an image and add some a11y control.
3. Save the project
4. Close the app
5. Restart the app from Xcode

**Expected result**
The project opens with all a11y controls, as well as the image.
**Actual result**
The project opens with all the a11y controls, but the image is missing, I need to add it again and again.

<img width=100% alt="Screenshot 2023-01-10 at 23 07 11" src="https://user-images.githubusercontent.com/29862757/211652264-4e7892fe-ff80-4f74-801f-8497dacd5f8c.png">
<img width=100% alt="Screenshot 2023-01-10 at 23 06 48" src="https://user-images.githubusercontent.com/29862757/211652255-3723b0fa-231a-4237-8c03-6c0d107e4f3a.png">

**Comment by prokhorovxo** [comment]
@akaDuality @drucelweisse guys check this pls. Have you encountered a similar bug? It reproduce on `main` branch.

**Comment by akaDuality** [comment]
Is far as I understand it the bug after #183

**Comment by akaDuality** [comment]
```
public convenience init(image: NSImage) {
    self.init(fileName: image.name() ?? Date().description)
    self.image = image
}
```

Setters don't called when is set from constructor

**Comment by akaDuality** [comment]
Going to fix it as part of #202
